nbviewer/frontpage.json

```diff
@@ -8,17 +8,17 @@
      "links":[
        {
          "text": "IPython",
-         "target": "github/ipython/ipython/blob/6.x/examples/IPython%20Kernel/Index.ipynb",
+         "target": "/github/ipython/ipython/blob/6.x/examples/IPython%20Kernel/Index.ipynb",
```

Comment:
Have you confirmed the absolute path works when nbviewer is not running as a JupyterHub service?

Reply:
Yes, you're right, it does work also when nbviewer is not running as a JupyterHub service. Sorry for forgetting to mention that.